docs/usage.md

```diff
@@ -98,76 +75,68 @@ except Exception as err:
-            output_path = output_dir + "out.jpg"
-            if os.path.exists(output_path):
-                os.remove(output_path)
-            c2pa_data = builder.sign_file(signer, data_dir + "A.jpg", output_dir + "out.jpg")
-            assert len(c2pa_data) > 0
-
-        reader = Reader.from_file(output_dir + "out.jpg")
-        print(reader.json())
-        manifest_store = json.loads(reader.json())
-        manifest = manifest_store["manifests"][manifest_store["active_manifest"]]
-        assert "python_test" in manifest["claim_generator"]
-        # Check custom title and format.
-        assert manifest["title"]== "My Title" 
-        assert manifest,["format"] == "image/jpeg"
-        # There should be no validation status errors.
-        assert manifest.get("validation_status") == None
-        assert manifest["ingredients"][0]["relationship"] == "parentOf"
-        assert manifest["ingredients"][0]["title"] == "A.jpg"
-
-    except Exception as e:
-        print("Failed to sign manifest store: " + str(e))
-        exit(1)
+try:
+    # Create a signer from certificate and key files
+    with open("path/to/cert.pem", "rb") as cert_file, open("path/to/key.pem", "rb") as key_file:
+        cert_data = cert_file.read()
+        key_data = key_file.read()
+
+        # Create signer info using cert and key info
+        signer_info = C2paSignerInfo(
+            alg=C2paSigningAlg.PS256,
+            cert=cert_data,
+            key=key_data,
+            timestamp_url="http://timestamp.digicert.com"
+        )
+
+        # Create signer using the defined SignerInfo
+        signer = Signer.from_info(signer_info)
+
+        # Create builder with manifest and add ingredients
+        with Builder(manifest_json) as builder:
+            # Add any ingredients if needed
+            with open("path/to/ingredient.jpg", "rb") as ingredient_file:
+                ingredient_json = json.dumps({"title": "Ingredient Image"})
+                builder.add_ingredient(ingredient_json, "image/jpeg", ingredient_file)
+
+            # Sign the file
+            with open("path/to/source.jpg", "rb") as source_file, open("path/to/output.jpg", "wb") as dest_file:
+                manifest_bytes = builder.sign(signer, "image/jpeg", source_file, dest_file)
+
+            # Verify the signed file
+        with Reader("path/to/output.jpg") as reader:
+            manifest_store = json.loads(reader.json())
+            active_manifest = manifest_store["manifests"][manifest_store["active_manifest"]]
+            print("Signed manifest:", active_manifest)
+
+except Exception as e:
+    print("Failed to sign manifest store: " + str(e))
 ```
 
 ## Stream-based operation
 
 Instead of working with files, you can read, validate, and add a signed manifest to streamed data.  This example code does the same thing as the file-based example.
 
-### Read and validate C2PA data
+### Read and validate C2PA data using streams
```

Comment:
To avoid duplicate titles in the md file. There probably are better wordings.

Reply:
Duplicate titles are not a problem.  But this change is fine--just wordier.

Reply:
Duplicate titles are flagged by my `.md` linter ;) .